stable/locust/values.yaml

```diff
@@ -36,11 +37,105 @@ loadtest:
   # loadtest.locustCmd -- The command to run Locust
   locustCmd: "/opt/venv/bin/locust"
 
+locustfiles:
+  # locustfiles.mountPath -- the path of the locustfiles (without trailing backslash)
+  mountPath: "/mnt/locust"
+  # locustfiles.requirements -- Path to a file containing requirements to install
+  requirements: ~
+  gitSync:
```

Comment:
I think you can simply remove a lot of they keys in here to make it simpler, e.g. containerName, uid, securityContext, securityContexts, containerLifecycleHooks, extraVolumeMounts etc.

Do we really need to set all these just to get git-sync working? I would say just leave all these as k8s defaults if not